.editorconfig

```diff
@@ -1,4 +1,4 @@
-root = true
+#root = true
```

Comment:
Uhh, i commented it out to be able setting the indent_size to 3 (by having a higher order editorconfig), which is what i prefer. I will revert it if you don't agree with it.